carTelemetry2val/carTelemetry_Kuksa2HONO.py

```diff
@@ -0,0 +1,222 @@
+#! /usr/bin/env python
+#  Copyright (c) 2020 Contributors to the Eclipse Foundation
+#
+#  See the NOTICE file(s) distributed with this work for additional
+#  information regarding copyright ownership.
+#
+#  This program and the accompanying materials are made available under the
+#  terms of the Eclipse Public License 2.0 which is available at
+#  http://www.eclipse.org/legal/epl-2.0
+#
+#  SPDX-License-Identifier: EPL-2.0
+
+from __future__ import print_function, unicode_literals
+
+import os, sys
+import signal
+import json
+import threading
+import time
+import configparser
+import kuksa_viss_client as kuksaVal
+from kuksa_viss_client import KuksaClientThread
+
+import requests
+import paho.mqtt.publish as publish
+from proton.handlers import MessagingHandler
+from proton.reactor import Container
+from requests.auth import HTTPBasicAuth
+
+registryIp = "hono.eclipseprojects.io"
+httpAdapterIp = "hono.eclipseprojects.io"
+mqttAdapterIp = "hono.eclipseprojects.io"
+amqpNetworkIp = "hono.eclipseprojects.io"
+
+# Register Tenant
+tenant = requests.post(f'http://{registryIp}:28080/v1/tenants').json()
+tenantId = tenant["id"]
+
+print(f'Registered tenant {tenantId}')
+
+# Add Device to Tenant
+device = requests.post(f'http://{registryIp}:28080/v1/devices/{tenantId}').json()
+deviceId = device["id"]
+
+print(f'Registered device {deviceId}')
+
+# Set Device Password
+devicePassword = "my-secret-password"
+
+code = requests.put(f'http://{registryIp}:28080/v1/credentials/{tenantId}/{deviceId}',
+                    headers={'content-type': 'application/json'},
+                    data=json.dumps(
+                        [{"type": "hashed-password", "auth-id": deviceId, "secrets": [{"pwd-plain": devicePassword}]}]))
+
+if code.status_code == 204:
+    print("Password is set!")
+else:
+    print("Unable to set Password")
+
+# Now we can start the client application
+print("We could use the Hono Client now...")
+print()
+cmd = f'java -jar hono-cli-*-exec.jar --hono.client.host={amqpNetworkIp} ' \
+    f'--hono.client.port=15672 --hono.client.username=consumer@HONO ' \
+    f'--hono.client.password=verysecret --spring.profiles.active=receiver ' \
```

Comment:
I think it would be good to have a README.md file in this directory telling what the feeder does and how it can be used as well as the current status of the feeder. As it includes hard-coded dummy passwords I assume it at least currently shall be considered as prototype/designbase, i.e. anyone intending to use it for production purposes must refactor it so that the password is not hardcoded